README.md

```diff
@@ -21,3 +21,22 @@ GITHUB_RUN_NUMBER
 ```
 
 and then runs the [gke-start.sh](scripts/gke-start.sh)
+
+## Setup GKE cluster
+
+```bash
+gcloud container clusters create "${GKE_CLUSTER_NAME}" --project="${GKE_PROJECT_ID}" --machine-type="${GKE_CLUSTER_TYPE}" --num-nodes="${GKE_CLUSTER_NUM_NODES}" --zone="${GKE_CLUSTER_ZONE}" -q
+gcloud container clusters get-credentials "${GKE_CLUSTER_NAME}" --project="${GKE_PROJECT_ID}" --zone="${GKE_CLUSTER_ZONE}"
+gcloud compute firewall-rules create allow-proxy-nsm --action ALLOW --rules tcp:80 --project="${GKE_PROJECT_ID}"
```

Comment:
Do we really need to allow *all* port 80 traffic in?  Of could we do something more constrainted?

Reply:
I found that we don't need this for single cluster testing.